fastlane/Fastfile

```diff
@@ -101,16 +101,11 @@ platform :ios do
 
   lane :set_up_code_signing do |readonly: true|
     set_up_code_signing_app_store(readonly: readonly)
-    set_up_code_signing_deveploment(readonly: readonly)
   end
 
   lane :set_up_code_signing_app_store do |readonly: true|
     set_up_code_signing(type: 'appstore', readonly: readonly)
   end
```

Comment:
I could have refined this and removed the two-step abstraction, consolidating everything in the `set_up_code_singing` lane.

I decided against it to 1) save time 2) leaving the code like this doesn't make it much more complex and it has the advantage of being already layered in case we'll add additional code signing types in the future.